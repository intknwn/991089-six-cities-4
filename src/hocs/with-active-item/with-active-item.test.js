import React from "react";
import renderer from "react-test-renderer";
import withActiveItem from "./with-active-item.jsx";

const MockComponent = () => (
  <div></div>
);

const MockComponentWrapped = withActiveItem(MockComponent);

it(`withActiveItem render`, () => {
  const tree = renderer.create((
    <MockComponentWrapped onActiveItemSet={() => {}} />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
