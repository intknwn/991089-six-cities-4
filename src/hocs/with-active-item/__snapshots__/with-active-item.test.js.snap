// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveItem render 1`] = `<div />`;
